src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -404,7 +326,8 @@ else if (!userObj.isFake())
             // Fire update events
             updateUser(userObj, user);
         }
-        if (!fake && modifyCache)
+
+        if (newUser)
             getJDA().getEventCache().playbackCache(EventCache.Type.USER, id);
```

Comment:
Possible that if someone uses the cache for something to do `getUserById` or a scan over `getUsers` that this user won't be in the cache when these events are played back.

Reply:
This shouldn't even be here since the events will just be added to cache again. Will remove since this is now done by updateMemberCache.